src/client/swagger/managedFunctions.yml

```diff
@@ -298,27 +365,52 @@ components:
           readOnly: true
         url:
           type: string
-          description: trigger endpoint address
+          description: invocation endpoint address
         updatedAt:
           type: string
           format: date-time
           readOnly: true
       required: [name, orgID, script]
     Functions:
+      type: array
+      items:
+        $ref: '#/components/schemas/Function'
+    FunctionHTTPResponseData:
+      description: The data sent to end user when a function is invoked using http. User defined and dynamic
       type: object
+    FunctionHTTPResponse:
+      description: The full response sent to end user when a function is invoked using http
       properties:
-        functions:
-          type: array
-          items:
-            $ref: '#/components/schemas/Function'
+        type:
+          type: string
+          enum: [http]
+        data:
+          $ref: '#/components/schemas/FunctionHTTPResponseData'
+        dataType:
+          type: string
+          enum: [json]
+        headers:
+          type: object
+          properties:
+            contenttype:
+              type: string
+              enum: [application/json]
+            xtestresponse:
+              type: string
+              enum: [unspecified]
```

Comment:
these properties are supposed to be `content-type` and `x-test-response` with dashes!! but oats breaks on these property names... so I am entering them as they are for now until oats can be updated.

Reply:
https://github.com/influxdata/idpe/issues/9908

Reply:
I actually decided to remove these, since I'd rather have no definition rather than a definitely wrong one.